src/main/java/nl/tudelft/contextproject/ContextTFP.java

```diff
@@ -22,7 +22,7 @@
 import java.util.ArrayList;
 
 /**
- * This is the main file for the contextproject of Team Free Pizza.
+ * This is the main file for the Multi-Media Contextproject of Team Free Pizza.
  * The main purpose of this project is to allow PolyCast Productions B.V. to
  * easily control their cameras and to improve their digital environment.
  * 
```

Comment:
@bvobart It is? Line 31 is: `<p>This file shoud be used to initialize the program.`